scripts/gha_unit_tests_collector.py

```diff
@@ -13,29 +13,27 @@ def get_all_unit_test_shards():
     return unit_test_workflow_shards
 
 
-def get_modules_except_cms():
+def get_unit_test_modules(module_name="lms"):
```

Comment:
If you always call the function with an explicit argument, it's better to leave off the default.